docs/administrator.md

```diff
@@ -346,3 +346,19 @@ To read the latter log entry with the escaped characters rendered, view it
 in CLI with `echo -e`. Note that the resultant message will contain some
 noise because the `PodTemplate` used by the operator is yet to be updated
 with the default values used internally in Kubernetes.
+
+## Access to cloud resources from clusters in non cloud environment
+
+To access cloud resources like S3 from a cluster in a bare metal setup you can use
+`additional_secret_mount` and `additional_secret_mount_path` config parameters.
+With this you can provision cloud credentials to the containers in the pods of the StatefulSet.
+This works this way that it mounts a volume from the given secret in the pod and this can
+then accessed in the container over the configured mount path. Via [Custum Pod Environment Variables](## Custom Pod Environment Variables)
```

Comment:
... of the StatefulSet.
A volume is mounted from the given secret in the pod and can then be accessed in the container over the configured mount path. Via [Custom Pod Environment Variables](## Custom Pod Environment Variables) ... 

btw: link doesn't work for me

Reply:
fixed